File: app/src/main/java/com/rimir/linky/util/URLSUtils.java
package com.rimir.linky.util;

public class URLSUtils {

    public static String getExtension(String url) {
        try {
            String fileExtension = url.substring(url.lastIndexOf("."));
            return fileExtension;
        } catch (Exception e) {
            e.printStackTrace();
            return "error";
        }
    }
}

File: app/src/main/java/com/rimir/linky/util/URLSUtils.kt
package com.rimir.linky.util

import java.lang.Exception

object URLSUtils {
    fun getExtension(url: String): String {
        return try {
            url.substring(url.lastIndexOf("."))
        } catch (e: Exception) {
            e.printStackTrace()
            "error"
        }
    }
}